addons/l10n_ar_withholding/wizards/account_payment_register.py

```diff
@@ -0,0 +1,144 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import models, fields, api, Command, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = 'account.payment.register'
+
+    l10n_ar_withholding_ids = fields.One2many(
+        'l10n_ar.payment.register.withholding', 'payment_register_id', string="Withholdings",
+        compute='_compute_withholdings', readonly=False, store=True)
+    l10n_ar_net_amount = fields.Monetary(compute='_compute_l10n_ar_net_amount', readonly=True,  help="Net amount after withholdings")
+
+    @api.depends('line_ids', 'can_group_payments', 'group_payment')
+    def _compute_withholdings(self):
+        supplier_recs = self.filtered(lambda x: x.partner_type == 'supplier' and (not x.can_group_payments or (x.can_group_payments and x.group_payment)))
+        for rec in supplier_recs:
+            taxes = rec._get_withholding_tax()
+            rec.l10n_ar_withholding_ids = [Command.clear()] + [Command.create({'tax_id': x.id, 'base_amount': 0}) for x in taxes]
+        (self - supplier_recs).l10n_ar_withholding_ids = False
+
+    # @api.depends('l10n_latam_check_id', 'l10n_ar_withholding_ids.amount')
```

Comment:
please finish this new implementation

Reply:
add also the chanes of the payme register done by this PR on a separte commit https://github.com/odoo/odoo/pull/130305/files#diff-a006380282c121a102360b6b57d8f0d82a3d6c63e9e400992b0427838b8784ef

Reply:
DONE!